Docker/Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV LANG="en_US.utf8" APP_NAME="watcher3" IMG_NAME="watcher3" PATH="/opt/watcher
 
 RUN apk add --no-cache bash curl git nano vim ca-certificates python3 su-exec
 
-COPY . /opt/$APP_NAME
+RUN mkdir /opt/$APP_NAME && cd /opt/$APP_NAME && git clone https://github.com/barbequesauce/Watcher3 .
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use WORKDIR to switch to a directory](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=5404263)